New onnx - failed workflow run

From our [Friday runner](https://github.com/microsoft/hummingbird/actions/runs/4897649694), something is wrong with the `skl2onnx` install

```
        if not utils.skl2onnx_installed():
>           raise RuntimeError('skl2onnx is not installed. Please install skl2onnx to use this feature.')
E           RuntimeError: skl2onnx is not installed. Please install skl2onnx to use this feature.
```

--- a/setup.py
+++ b/setup.py
@@ -30,10 +30,9 @@ install_requires = [
     "protobuf>=3.20.2",
 ]
 onnx_requires = [
-    "onnx<=1.13.0",
     "onnxruntime>=1.0.0",
     "onnxmltools>=1.6.0,<=1.11.0",
-    "skl2onnx>=1.7.0,<=1.13.0",
+    "skl2onnx>=1.7.0",
 ]
 extra_requires = [
     # The need each for these depends on which libraries you plan to convert from

